TestComponentCSharp/ManualProjectionTestClasses.cpp

```diff
@@ -0,0 +1,104 @@
+#include "pch.h"
+#include "ManualProjectionTestClasses.h"
+#include "IBindableIteratorTest.g.cpp"
+#include "IDictionaryTest.g.cpp"
+#include "IDisposableTest.g.cpp"
+
+namespace winrt
```

Comment:
consider moving from here and class.cpp into pch.h, with a TODO to replace with latest cppwinrt

Reply:
TestWinRT already has a test for IDictionary but it did not test the APIs (Didn't call all the methods on it). I made this component initially but later added all the  API tests in the existing test in TestComponent.cs

Reply:
your response seems to be unrelated to my question, which is about single_threaded_vector_view, single_threaded_map_view